activities/communication/basic-tweet-template.md

```diff
@@ -0,0 +1,19 @@
+---
+type: resource
+---
+
+# Basic tweet template
+
+Most of our tweets follow this format:
+
+> Audience hook question or statement - what's the most interesting take on or valuable aspect of the thing we're pitching? Call to action: join us/read/check out/verb the precisely-described-thing (if relevant: at date, time UTC and place. Joining details/find out more: URL
+
+Our tweets are often accompanied by relevant images or emoji.
+
+## When to use this template
+
+When suggesting something for social sharing, use this template to create  a draft tweet. Also consider images, gifs or emoji. Note that Twitter's algorithm prioritizes tweets with images or video.
```

Comment:
I think I see an extra space between "create" and the "a" (warm regards, the nit picker)

Reply:
Done in a separate PR!